MIGRATING.md

```diff
@@ -1,5 +1,48 @@
 ## Migration Guides
 
+### Migrating from versions < 19.0.0
+
+* Deprecates `publishableKey` and `stripeAccount` properties of `STPPaymentConfiguration`. 
+  * If you used `STPPaymentConfiguration.sharedConfiguration` to set `publishableKey` and/or `stripeAccount`, use `STPAPIClient.sharedClient` instead. 
+  * If you passed a STPPaymentConfiguration instance to an SDK component, you should instead create an STPAPIClient, set publishableKey on it, and set the SDK component's APIClient property.
+* The SDK now uses `STPAPIClient.sharedClient` to make API requests by default.
+
+This changes the behavior of the following classes, which previously used API client instances configured from `STPPaymentConfiguration.shared`: `STPCustomerContext`, `STPPaymentOptionsViewController`, `STPAddCardViewController`, `STPPaymentContext`, `STPPinManagementService`, `STPPushProvisioningContext`. 
+
+You are affected by this change if:
+
+1. You set `stripeAccount`
+2. You use one of these classes
+3. `STPPaymentConfiguration.shared.stripeAccount != STPAPIClient.shared.stripeAccount`
+
+If these are all true, you must update your integration!  The SDK used to send the `STPPaymentConfiguration.shared.stripeAccount`, and will now send `STPAPIClient.shared.stripeAccount`.  
+
+For example, if you are a Connect user who stores Payment Methods on your platform, but clones PaymentMethods to a connected count and create direct charges on that connected account ie if:
+
+1. You never set `STPPaymentConfiguration.shared.stripeAccount`
+2. You set `STPAPIClient.shared.stripeAccount`
+
+We recommend you do the following:
+
```

Comment:
Maybe rewrite as:

For example, if you only want to use a Connected account for a single transaction, your integration could look like:

Reply:
That sounds different from the use case described.  These users want to use a Connected account for all transactions, if transactions means payments, but they want to retrieve Customers and their PMs on the platform.

Reply:
okay, as written it was confusing to me but I think you understand the use case better :)